### PR TITLE
fixed pop movie item layout

### PR DESCRIPTION
**Pull request by alishechka**
correct base

### CODE REVIEW

File: app/src/main/res/layout/popular_movie_item.xml [hunk 1]
```diff
@@ -2,39 +2,47 @@
...
+        android:padding="@dimen/padding_5dp">
 
         <TextView
             android:id="@+id/tv_movieName"
-            android:layout_width="wrap_content"
+            android:layout_width="0dp"
             android:layout_height="wrap_content"
-            android:layout_marginStart="@dimen/tv_margin_start"
+            android:layout_margin="@dimen/padding_5dp"
+            android:gravity="center"
```
Comment (EMHankombo) [inline app/src/main/res/layout/popular_movie_item.xml:22]: Is this needed for constraint layout?

File: app/src/main/res/layout/popular_movie_item.xml [hunk 2]
```diff
@@ -2,39 +2,47 @@
 <androidx.cardview.widget.CardView xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
-    android:orientation="vertical"
     android:layout_width="match_parent"
     android:layout_height="@dimen/card_height"
-    android:layout_marginBottom="@dimen/card_margin_bot">
+    android:layout_margin="@dimen/marginTop"
+    android:orientation="vertical"
```
Comment (EMHankombo) [inline app/src/main/res/layout/popular_movie_item.xml:8]: Don't think this is needed

**Review by EMHankombo — CHANGES_REQUESTED** [review]
Adress minor fixes

File: app/src/main/res/layout/popular_movie_item.xml (hunk 1 above)
Comment (alishechka) [reply app/src/main/res/layout/popular_movie_item.xml:22]: android:gravity="center" removed

File: app/src/main/res/layout/popular_movie_item.xml (hunk 2 above)
Comment (alishechka) [reply app/src/main/res/layout/popular_movie_item.xml:8]: android:orientation="vertical" removed

**Pull request closed without merging**